components/_dropdowns.scss

```diff
@@ -0,0 +1,38 @@
+$dropdown-content-z-index: 1;
+
+.wds-dropdown {
+	display: inline-block;
+	position: relative;
+
+	&.wds-is-active {
+		.wds-dropdown-content {
+			display: inline-block;
+		}
+
+		.wds-dropdown-toggle-chevron {
+			transform: rotate(180deg);
+		}
+	}
+}
+
+.wds-dropdown-toggle {
+	position: relative;
+	// dropdown content has shadow that should be hidden below the toggle
+	z-index: $dropdown-content-z-index + 1;
+}
+
+.wds-dropdown-content {
+	background-color: $color-white;
+	box-shadow: 0 0 10px 0 rgba(0, 0, 0, .2);
```

Comment:
Let's address it later when we decide what to do with `$color-black` which isn't black.